lib/etl/match_augment.ts

```diff
@@ -0,0 +1,20 @@
+import {queryMatches, getFullMatchDetailsFromAPI, writeGamerMatchesToDatabase} from "../model/matches";
```

Comment:
@fjaxyu this job isn't quite done yet. I can't schedule this job until we get the data model concerns figured out.

Reply:
What do you think about moving these model stuffs into the components?